### PR TITLE
Catch up with Embulk v0.10 API/SPI

### PR DESCRIPTION
**Pull request by dmikurube**
Catching up with Embulk v0.10 API/SPI. See https://dev.embulk.org/topics/get-ready-for-v0.11-and-v1.0.html for the details.

I've already had #48, but Embulk and util libraries had some more updates. It should be the last catch-up before Embulk v0.11.0...

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -76,22 +73,46 @@ embulkPlugin {
...
+                    connection = "scm:git:git://github.com/civitaspo/embulk-filter-expand_json.git"
+                    developerConnection = "scm:git:git@github.com:civitaspo/embulk-filter-expand_json.git"
+                    url = "https://github.com/civitaspo/embulk-filter-expand_json"
+                }
+            }
         }
     }
+
     repositories {
         // TODO: Configure some public Maven repo(s).
```
Comment (dmikurube) [inline build.gradle:117]: If you're willing to release its JARs to Maven Central, this part would be like below :

```suggestion
        maven {  // publishMavenPublicationToMavenCentralRepository
            name = "mavenCentral"
            if (project.version.endsWith("-SNAPSHOT")) {
                url "https://oss.sonatype.org/content/repositories/snapshots"
            } else {
                url "https://oss.sonatype.org/service/local/staging/deploy/maven2"
            }

            credentials {
                username = project.hasProperty("ossrhUsername") ? ossrhUsername : ""
                password = project.hasProperty("ossrhPassword") ? ossrhPassword : ""
            }
        }
```

File: build.gradle [hunk 2]
```diff
@@ -76,22 +73,46 @@ embulkPlugin {
...
+                licenses {
+                    license {
+                        // http://central.sonatype.org/pages/requirements.html#license-information
+                        name = "MIT License"
+                        url = "https://www.opensource.org/licenses/mit-license.php"
+                    }
+                }
+
+                developers {
+                    // TODO: Set <developers> if released for Maven Central.
```
Comment (dmikurube) [inline build.gradle:104]: If you're willing to release its JARs to Maven Central, it needs at least one `developer` configured. For example,

```suggestion
                    developer {
                        name = "Civitaspo"
                        email = "civitaspo@gmail.com"
                    }
```

**Review by dmikurube — COMMENTED** [review]
@civitaspo Could you have a look when you have time?

You may want to update the version number again (`0.5.3` or `0.6.0`?), in addition.

**Review by civitaspo — APPROVED** [review]

**Comment by civitaspo** [comment]
shipped as v0.5.3 https://github.com/civitaspo/embulk-filter-expand_json/releases/tag/v0.5.3

**Pull request merged**